WordPress/build.gradle

```diff
@@ -57,7 +57,7 @@ android {
         } else {
             versionName "alpha-237"
         }
-        versionCode 903
+        versionCode 902000
```

Comment:
This doesn't work here. We need a different versionCode for the Zalpha than the one used by the beta. If we use the same code, it will fail when the second build is uploaded to the store.